infor-design/design-system
# 205 Tokens 2.0 - Switch the token process to use amzn/style-dictionary

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Theo wasn't quite working the way we'd hope with theming, so we switched over to use https://github.com/amzn/style-dictionary as this is more `cascading` to our liking.

**Related github/jira issue (required)**:
Fixes #205 
Fixes #202 

**Steps necessary to review your pull request (required)**:

To check for vulnerabilities:
1. Run `npm audit`
1. See results:
    ```
    === npm audit security report ===

    found 0 vulnerabilities
    in XXXX scanned packages
    ```

To test tokens:
1. Run `npm run build`
2. Example the contents of `dist/web/tokens` and verify that we have the correct outputs/contents.
